src/main/java/uk/gov/companieshouse/config/CompanyNumberConfiguration.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.companieshouse.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class CompanyNumberConfiguration {
+    @Value("${companyNumber.prefixes}")
```

Comment:
At the moment we only have one property, we could just inject this into the validator?